skglm/solvers/prox_newton_solver.py

```diff
@@ -0,0 +1,221 @@
+import numpy as np
+from scipy import sparse
+from numba import njit
+from skglm.datafits import Logistic, Logistic_32
+from skglm.solvers.common import construct_grad
+
+
+def prox_newton_solver(
+    X, y, datafit, penalty, w, Xw, max_iter=50, max_epochs=1000, max_backtrack=10,
+    min_pn_cd_epochs=2, max_pn_cd_epochs=10, p0=10, tol=1e-4, verbose=0
+):
```

Comment:
cosmetics, bring this to L10

Reply:
flake8 raise a `E125 continuation line` warning

Reply:
A tab must be added, 
suggestion for the whole function signature

```suggestion
def prox_newton_solver(
        X, y, datafit, penalty, w, Xw, max_iter=50, max_epochs=1000, max_backtrack=10,
        min_pn_cd_epochs=2, max_pn_cd_epochs=10, p0=10, tol=1e-4, verbose=0):
```